homeassistant/components/subaru/entity.py

```diff
@@ -48,13 +43,3 @@ def device_info(self):
     def available(self):
```

Comment:
I think you can also remove this when inheriting from `CoordinatorEntity`

Reply:
Removed.